src/doctest.jl

```diff
@@ -59,13 +59,18 @@ manual pages can be disabled if `source` is set to `nothing`.
     develop mode and commit any staged changes. You should also review all the changes made
     by `doctest` before committing them, as there may be edge cases when the automatic
     fixing fails.
+
+Select other keyword arguments, e.g., `plugins`, are internally forwarded to
+[`makedocs`](@ref). These should only be used as directed, e.g., in the documentation of a
+third-party plugin.
 """
 function doctest(
         source::Union{AbstractString,Nothing},
         modules::AbstractVector{Module};
         fix = false,
         testset = "Doctests",
         doctestfilters = Regex[],
+        kwargs...
```

Comment:
Hmm. I feel like we should be conservative here and only pass `plugins`, not all of them. If at some point we discover that we should be passing others, then we add them as needed.

Reply:
Sure! I was on the fence about this one, but just passing `plugins` should be totally fine.

Reply:
Ok, force-pushed a new version